gradle.properties

```diff
@@ -23,5 +23,5 @@ android.defaults.buildfeatures.resvalues=false
 
 # Release - Module-specific versions
 coreVersion=5.2.0
-androidVersion=3.27.1
+androidVersion=3.27.2
```

Comment:
this version will core core=5.2.0 which is being published now and fixes the issue https://github.com/PostHog/posthog-android/actions/runs/19899349776/job/57038427276